manifests/dev/vanilla/vsphere-csi-driver.yaml

```diff
@@ -141,7 +141,7 @@ spec:
       dnsPolicy: "Default"
       containers:
         - name: csi-attacher
-          image: quay.io/k8scsi/csi-attacher:v3.1.0
+          image: quay.io/k8scsi/csi-attacher:v3.2.0
```

Comment:
Can you add new parameters for qps and burst? https://github.com/kubernetes-csi/external-attacher/blob/release-3.2/CHANGELOG/CHANGELOG-3.2.md

Reply:
You mean the below params ?
```
            - "--kube-api-qps=100"
            - "--kube-api-burst=100"
```

Reply:
can you also mention why do we need it now ?
cc: @RaunakShah